content/en/tutorials/migrate_gcp_122.md

```diff
@@ -0,0 +1,18 @@
+# Upgrading Google Kubernetes Engine Version
```

Comment:
missing some headers https://github.com/run-x/opta-docs/blob/main/content/en/tutorials/iam.md?plain=1#L1-L8

Reply:
> 

removed

Reply:
we need these headers:

ex:
```
---
title: "Changing Domains"
linkTitle: "Changing Domains"
date: 2022-01-03
draft: false
description: >
  How to change domains for your Opta environment.
---
```
see other files for example